iOS/ModelRunner/AppDelegate.h

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
FYI, we have mobile examples (including ios) under the mobile directory.
perhaps these examples should use the iOS packages too.

Reply:
You mean, combine some of the [mobile/examples](https://github.com/microsoft/onnxruntime-inference-examples/tree/main/mobile/examples) and the iOS dir?

Reply:
and perhaps adapt these examples to use the onnxruntime-mobile-c or onnxruntime-mobile-objc packages

Reply:
I will create work items for them.